community/releases/roadmap.md

```diff
@@ -91,18 +22,35 @@ ones.
 
 ### Kubernetes
 
-* Integrate with third-party docker builders in the "bake" stage
 * Simplify process for registering & deregistering new accounts by allowing GKE
   credentials to manage the list of clusters configured in Spinnaker. This can be
   extended to AKS & EKS as well.
+* Round out integration test coverage.
+
+### Artifacts
+
+* Determine and implement contracts for CI systems supplying artifacts to
+  Spinnaker. This will improve artifact traceability in Spinnaker, and surface
+  relevant CI context on Pipelines and infrastructure.
+* Round out integration test coverage.
+
+### Entity tags
+
+* Address gaps in OSS entity-tag support, including installation,
+  configuration, and provider support.
+
+### Halyard
+
+* Begin planning a Spinnaker-integrated "Halyard UI" to allow users (not
```

Comment:
Is there any way we can recommend interested users to get involved on this topic?

Reply:
:+1: will add a callout